apps/ledger-live-mobile/src/components/BleDevicePairingFlow/BleDevicesScanning.tsx

```diff
@@ -30,6 +31,10 @@ export type BleDevicesScanningProps = {
 /**
  * Runs a BLE scan and list seen devices
  *
+ * This components should be wrapped in a RequiresBLE component.
+ * This is the case in the BleDevicePairingFlow component.
+ * If this is not the case, some BLE and locations errors are handled, but not as well as with RequiresBLE.
+ *
```

Comment:
one idea: you could enforce that by creating a simple context
```
const RequiresBLEContext = React.createContext({ wrappedInRequiresBLE: false });
```
In `RequiresBLE` you wrap the returned node with:
```
<RequiresBLEContext.Provider value={{ wrappedInRequiresBLE: true })>
  {/* ... */}
</RequiresBLEContext.Provider>
```

and in this component (and any others with similar requirements):
```
const { wrappedInRequiresBLE } = useContext(RequiresBLEContext);
if (!wrappedInRequiresBLE)
  console.warn("BleDevicesScanning should have a RequiresBLE component in his ancestors to ensure proper handling of BLE and locations errors");
```